components/item/AwesomeSwiper.vue

```diff
@@ -10,12 +10,23 @@
       v-for="(item, index) in displayableMedia"
       :key="index"
     >
+      <div
+        v-if="singleMediaResource"
+        class="container h-100"
```

Comment:
I am not sure if it is due to this `h-100`, but on mobile, single images don't keep their aspect ratio. I think they should. Try with item, /2020738/3DICONS_8432, the image is a square, but on mobile not.